fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/detail/event/election/ElectionResultFragmentTest.java

```diff
@@ -112,19 +139,24 @@ protected void before() throws UnknownLaoException {
           ElectionResultFragment.class, ElectionResultFragment::newInstance);
 
   @Before
-  public void setUpViewModel() {
+  public void setUp() {
     fragmentRule
         .getScenario()
         .onFragment(
             fragment -> {
               FragmentActivity fragmentActivity = fragment.requireActivity();
               LaoDetailViewModel viewModel = LaoDetailActivity.obtainViewModel(fragmentActivity);
               viewModel.setCurrentLao(new LaoView(LAO));
-              viewModel.setCurrentElection(election);
+              viewModel.setCurrentElection(ELECTION.getId());
```

Comment:
I look forward to the day where we'll part ways with those. Horrible for testing